### ISSUE TITLE
Documentation of plugin schema outdated

### ISSUE THREAD

**Issue by marckhouzam** [Kong/kong#5471]
### Summary

After installing Kong 1.4.2 and using the latest version of Konga (https://github.com/pantsel/konga), I'm getting the following type of errors:
```
2020/01/20 09:27:08 [warn] 47881#0: *2747037 [kong] plugins.lua:174 DEPRECATED: /plugins/schema/:name endpoint is deprecated, please use /schemas/plugins/:name instead., client: 127.0.0.1, server: kong_admin, request: "GET /plugins/schema/cors?size=1000 HTTP/1.1", host: "localhost:8001"
```
I wanted to propose a fix for Konga, but I cannot find any documentation of the new Kong API.  In fact the Admin API documentation still mentions `/plugins/schema` as the only way to obtain the plugin schema:
https://docs.konghq.com/1.4.x/admin-api/#retrieve-plugin-schema

**Comment by hbagdi** [comment]
This is indeed out of date.
Users are advised to use: `:8001/schemas/plugins/:plugin-name` as the new endpoint.

**Comment by hishamhm** [comment]
Thanks for the report! Will be fixed by #5583

**Comment by hishamhm** [comment]
#5583 is merged, so the problem will go away. Also sent a PR to the docs repo: https://github.com/Kong/docs.konghq.com/pull/1805
